tests/baselines/reference/jsxEmptyExpressionNotCountedAsChild(jsx=react-jsx).js

```diff
@@ -21,8 +21,6 @@ const element = (
 "use strict";
 exports.__esModule = true;
 var jsx_runtime_1 = require("react/jsx-runtime");
-/// <reference path="react16.d.ts" />

```

Comment:
from what I understand this actually got fixed because the default of `importsNotUsedAsValues` is `remove` and nothing from `'react'` module is used here in the emitted output